src/transport/transport.cc

```diff
@@ -45,17 +45,38 @@
 
 #include "reaction/isotherm.hh" // SorptionType enum
 
+
+FLOW123D_FORCE_LINK_IN_CHILD(convectionTransport);
+
+
 namespace IT = Input::Type;
 
 
+const int ConvectionTransport::registrar =
+		Input::register_class< ConvectionTransport, Mesh &, const Input::Record >("Convection_FV") +
+		ConvectionTransport::get_input_type().size();
+
+const IT::Record &ConvectionTransport::get_input_type()
+{
+	return IT::Record("Convection_FV", "Explicit in time finite volume method for solute transport.")
+			.derive_from(ConcentrationTransportBase::get_input_type())
+			.declare_key("input_fields", IT::Array(
+					IT::Record("TransportOperatorSplitting_Data", FieldCommon::field_descriptor_record_decsription("TransportOperatorSplitting_Data") )
```

Comment:
Since we do add any particular keys as in the DarcyFlow, the EqData().make_field_descriptor_type("Transport_OS")
may be used directly.